avocado-plugins-vt.spec

```diff
@@ -153,8 +153,8 @@ Xunit output, among others.
 %if %{with_python3}
 %files -n python3-%{name}
 %defattr(-,root,root,-)
-%dir /etc/avocado
-%dir /etc/avocado/conf.d
+%dir %{_sysconfdir}/avocado
+%dir %{_sysconfdir}/avocado/conf.d
```

Comment:
Good point, would you please include the changelog entry for this? (use the same entry as in the second commit, and in the second commit add another item.

Reply:
Alright